src/Bridge/Graphql/Type/SchemaBuilder.php

```diff
@@ -335,6 +373,10 @@ private function getOperations(ResourceMetadata $resourceMetadata, bool $isQuery
                 continue;
             }
 
+            if (Request::METHOD_PATCH === $operation['method']) {
```

Comment:
You should probably whitelist methods here instead of blacklisting them, some other can be problematic (HEAD, OPTIONS...)

Reply:
Sure.